### PR TITLE
regenerate services in prep for 2022-10 release

### PR DESCRIPTION
**Pull request by bmc-msft**

### CODE REVIEW

File: services/mgmt/app/Cargo.toml [hunk 1]
```diff
@@ -29,9 +29,10 @@ env_logger = "0.9"
 all-features = true
 
 [features]
-default = ["package-2022-03", "enable_reqwest"]
+default = ["package-preview-2022-06", "enable_reqwest"]
```
Comment (cataggar) [inline services/mgmt/app/Cargo.toml:32]: This should remain on `package-2022-03` instead. I'll fix it with #1112.

File: services/mgmt/chaos/Cargo.toml [hunk 2]
```diff
@@ -29,9 +29,10 @@ env_logger = "0.9"
 all-features = true
 
 [features]
-default = ["package-2022-07-01-preview", "enable_reqwest"]
+default = ["package-preview-2022-10", "enable_reqwest"]
 enable_reqwest = ["azure_core/enable_reqwest"]
 enable_reqwest_rustls = ["azure_core/enable_reqwest_rustls"]
 no-default-tag = []
+"package-preview-2022-10" = []
```
Comment (cataggar) [inline services/mgmt/chaos/Cargo.toml:36]: They changed their naming convention here, but it worked out.

File: services/mgmt/confidentialledger/Cargo.toml [hunk 3]
```diff
@@ -29,10 +29,11 @@ env_logger = "0.9"
 all-features = true
 
 [features]
-default = ["package-2022-05-13", "enable_reqwest"]
+default = ["package-2022-09-08-preview", "enable_reqwest"]
```
Comment (cataggar) [inline services/mgmt/confidentialledger/Cargo.toml:32]: This should remain on `package-2022-05-13` for the default. I'll fix with #1112.

**Review by cataggar — APPROVED** [review]
This easier to review if the version change is a separate PR. After this PR is merged, I'd like to fix #1112 before a release if possible.

**Comment by bmc-msft** [comment]
@cataggar , note I undid the version bump, since I'll need to regenerate once I bump azure_core as part of the release process.

**Pull request merged**